lib/v2/aqara/news.js

```diff
@@ -3,51 +3,51 @@ const cheerio = require('cheerio');
 const { parseDate } = require('@/utils/parse-date');
 
 module.exports = async (ctx) => {
-    const limit = ctx.query.limit ? parseInt(ctx.query.limit) : 35;
+    const limit = ctx.query.limit ? parseInt(ctx.query.limit, 10) : 35;
 
     const rootUrl = 'https://www.aqara.com';
-    const currentUrl = `${rootUrl}/news`;
+    const currentUrl = new URL('news', rootUrl).href;
 
-    const response = await got({
-        method: 'get',
-        url: currentUrl,
-    });
+    const { data: response } = await got(currentUrl);
 
-    let items = response.data
-        .match(/parm.contentHerf = '(\d+)'/g)
+    const $ = cheerio.load(response);
+
+    let items = response
+        .match(/(parm\.newsTitle[\s\S]*?arr\.push\(parm\))/g)
```

Comment:
Nothing can be matched

Reply:
Testing locally works fine.

![aqara](https://github.com/DIYgod/RSSHub/assets/42264778/93bfaeaa-ab19-41ea-b106-fd8df5cc423f)

Reply:
Visiting https://www.aqara.cn/news using IP from CN will be redirected to `https://www.aqara.cn/news` while the rest will be redirected to `https://www.aqara.com/en/about-us/news/`

Reply:
WIP